presto-main/src/main/java/com/facebook/presto/execution/scheduler/SqlQueryScheduler.java

```diff
@@ -630,9 +633,15 @@ public void start()
 
     private List<SqlStageExecution> getStagesReadyForExecution()
     {
+        long runningPlanSections =
+                stream(forTree(StreamingPlanSection::getChildren).depthFirstPreOrder(sectionedPlan))
+                        .map(section -> getStageExecution(section.getPlan().getFragment().getId()).getState())
+                        .filter(state -> !state.isDone() && state != PLANNED)
```

Comment:
I feel we want to make `StageState` to have another boolean like `runningState`. But we don't need to do this in this PR